bridge_adaptivity/module/static/module/js/consumer.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+    $("#contributors .delete-contributor").on("click", function() {
+            var element = $(this);
+           $.ajax({
+              url: this.dataset["remove_url"],
+            }).done(function() {
+              element.closest("tr").remove();
+            }).fail(function (jqXHR, textStatus, errorThrown) {
+              console.log(textStatus + "(" + errorThrown + "): " + jqXHR.responseText.split('\n')[0]);
+              element.closest("tr").children("td.error").append("<p>We fix some problems. Please, try later.</p>");
```

Comment:
What does this text mean? Who are `We`? And what problems will be fixed? It should be some kind of general message, for example:
```
Cannot be removed, please try again later. If the problem will be procced contact with the Bridge admin.
```

Reply:
I changed this text.